net.crossfireatlas.crossfire.yaml

```diff
@@ -0,0 +1,26 @@
+app-id: net.crossfireatlas.crossfire
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: 
+command: net.crossfireatlas.crossfire.start.sh
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: 
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: Crossfire_client
+    buildsystem: simple
+    build-commands: 
+      - install -D download /app/bin/jxclient.jar
+      - install -D net.crossfireatlas.crossfire.start.sh /app/bin/net.crossfireatlas.crossfire.start.sh
+    sources:
+      - type: file
+        url: https://sourceforge.net/projects/crossfire/files/jxclient/jxclient.jar/download
```

Comment:
This URL looks unversioned. Is it on purpose?

Reply:
That is how it comes from sourceforge. So, yes.